<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Dialog</class>
    <widget class="QDialog" name="Dialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>409</width>
                <height>105</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Add Book</string>
        </property>
        <widget class="QLabel" name="label">
            <property name="geometry">
                <rect>
                    <x>20</x>
                    <y>10</y>
                    <width>46</width>
                    <height>20</height>
                </rect>
            </property>
            <property name="text">
                <string>Title:</string>
            </property>
        </widget>
        <widget class="QLabel" name="label_2">
            <property name="geometry">
                <rect>
                    <x>20</x>
                    <y>40</y>
                    <width>46</width>
                    <height>20</height>
                </rect>
            </property>
            <property name="text">
                <string>ISBN:</string>
            </property>
        </widget>
        <widget class="QLineEdit" name="title_input">
            <property name="geometry">
                <rect>
                    <x>70</x>
                    <y>10</y>
                    <width>331</width>
                    <height>20</height>
                </rect>
            </property>
        </widget>
        <widget class="QLineEdit" name="isbn_input">
            <property name="geometry">
                <rect>
                    <x>70</x>
                    <y>40</y>
                    <width>331</width>
                    <height>20</height>
                </rect>
            </property>
        </widget>
        <widget class="QDialogButtonBox" name="buttonBox">
            <property name="geometry">
                <rect>
                    <x>240</x>
                    <y>70</y>
                    <width>156</width>
                    <height>23</height>
                </rect>
            </property>
            <property name="orientation">
                <enum>Qt::Horizontal</enum>
            </property>
            <property name="standardButtons">
                <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
